src/main/resources/db/migration/ledgersyncagg/V1_4_2__add-agg-materialized-view.sql

```diff
@@ -26,15 +26,22 @@ GROUP BY ata.address, ata.stake_address, day
 ORDER BY day;
 ------------------------------------------------------------------------------------------------------------------------
 DROP TABLE IF EXISTS stake_tx_balance;
-CREATE MATERIALIZED VIEW IF NOT EXISTS stake_tx_balance AS
```

Comment:
Why did you remove the IF NOT EXISTS?

Reply:
I missed it when changing the MV query. Updated!